cleansweep/plugins/committees/templates/edit_committee_structure.html

```diff
@@ -6,9 +6,12 @@
   <ul class="breadcrumb breadcrumb-collapse">
     <li><a href="{{url_for('admin')}}">Admin Center</a></li>
     <li><a href="{{url_for('.committee_structures')}}">Committee Structures</a></li>
+    <li><a href="{{url_for('.committees_by_level', level=committee_type.place_type.short_name)}}"
+        >{{ committee_type.place_type.name }}</a></li>
+    <li><a href="{{ committee_type.url_for('.view_committee_structure') }}">{{ committee_type.name }}</a></li>
     <li></li>
   </ul>
-  <h2>Edit <a href="{{committee_type.url_for('.view_committee_structure')}}">{{committee_type.name}}</a></h2>
+  <h2>Edit</h2>
```

Comment:
I thought about this. It looks better if we add committee name in breadcrumbs.

![screen shot 2015-12-07 at 23 47 30](https://cloud.githubusercontent.com/assets/4872139/11635469/071b36d6-9d3d-11e5-8af5-03e59e284b4c.png)

I'll change it back if you say so.